src/extractor/guidance/sliproad_handler.cpp

```diff
@@ -689,31 +702,32 @@ bool SliproadHandler::nextIntersectionIsTooFarAway(const NodeID start, const Edg
     return accumulator.too_far_away;
 }
 
-bool SliproadHandler::isThroughStreet(const EdgeID from, const IntersectionView &intersection) const
-{
-    BOOST_ASSERT(from != SPECIAL_EDGEID);
-    BOOST_ASSERT(!intersection.empty());
+// bool SliproadHandler::isThroughStreet(const EdgeID from, const IntersectionView &intersection)
```

Comment:
Why is this commented out - do we still need it?

Reply:
Oh no, just forgot to remove it.